.github/workflows/api-tests-on-uv.yml

```diff
@@ -0,0 +1,83 @@
+name: Run Pytest on UV
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - api/**
+      - docker/**
+      - .github/workflows/api-tests.yml
+
+concurrency:
+  group: api-tests-on-uv-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: API Tests on uv
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+        # working-directory: ./api
+    strategy:
+      matrix:
+        python-version:
+          - "3.12"
```

Comment:
The `requires-python` setting in the pyproject.toml for the dify-api project specifies ">=3.11,<3.13". 

To ensure compatibility across all supported Python versions, it would be ideal to include Python `"3.11"` in the python-version list of the workflow.

Reply:
Hey , thanks for the review.
This PR is intented to show the possible impacts and it's not yet ready for merging as for now, as it is marked as draft status.
Here use the matrix but only `3.12` is to 1. pass the ci for most used python version , especially shipping Python 3.12 in Docker image for most end users; 2. saving ci test resources for this time.